content/docs/components-and-props.md

```diff
@@ -118,15 +118,15 @@ ReactDOM.render(
 );
 ```
 
-[](codepen://components-and-props/composing-components)
+[**Essayer sur CodePen**](codepen://components-and-props/composing-components)
 
-Typically, new React apps have a single `App` component at the very top. However, if you integrate React into an existing app, you might start bottom-up with a small component like `Button` and gradually work your way to the top of the view hierarchy.
+En règle générale, les nouvelles applications React n'ont un seul composant `App` en haut de leur hiérarchie.  Cependant, si vous intégrez React à une application existante, vous pouvez commencer par un petit composant comme `Button` et vous diriger progressivement vers le haut de la hiérarchie.
```

Comment:
```suggestion
En règle générale, les nouvelles applications React ont un seul composant `App` à la racine.  En revanche, si vous intégrez React à une application existante, vous pouvez commencer tout en bas par un petit composant comme `Button` et remonter progressivement la hiérarchie des vues.
```

“However” a de multiples sens, et suivant le contexte on optera plutôt pour une trad qu’une autre. Quand plusieurs trads se valent, on peut varier juste pour éviter d'être aussi répétitifs que la VO (Dan Abramov est capable de mettre 5 “However” en début de 5 phrases consécutives… 😉)

Parmi les traductions classiques : « Cependant », « Ceci dit », « Néanmoins », « En revanche », « (Et) pourtant », « Quoi qu’il en soit » (dont l'anglais exact est un peu trop formel et connoté un poil agressif pour de la doc tech: “be that as it may”).

Reply:
Haha j'avais jamais fais attention à cette utilisation massive de  "However". Raison de plus pour garder le Thésaurus pas trop loin :grin: